src/@commonsku/styles/ConfirmAddressPopup/ConfirmAddressContent.tsx

```diff
@@ -0,0 +1,216 @@
+          border: `2px solid ${selectedValue !== 'recommended' ? 'var(--color-neutrals-50)' : borderColor}`,
+          borderRadius: '5px',
+          padding: 14,
+        }}>
+          {!loading && recommendedAddress && <>
+            <Col xs sm={2} style={{ minHeight: '40px' }}>
+              <LabeledRadio
+                name="recommended-checkbox"
+                label=""
+                checked={selectedValue === 'recommended'}
+                onChange={(e) => {
+                  e?.preventDefault?.();
+                  setSelectedValue(recommendedAddress ? 'recommended' : 'entered');
+                }}
+              />
+            </Col>
+            <Col xs sm={10}>
+              <Typography.Text bold fontSize={'18px'} style={{ marginBottom: '4px' }}>Recommended</Typography.Text>
+              {loading ? <div style={{ textAlign: 'center' }}>
+                  <Loading />
+              </div> : <>
+                <Typography.Text noBottom>{recommendedAddress?.address_line_1 || ''}</Typography.Text>
+                <Typography.Text noBottom>{getAddressCityAndState(recommendedAddress || {})}</Typography.Text>
+                <Typography.Text noBottom>{getAddressPostalCodeAndCountry(recommendedAddress || {})}</Typography.Text>
+              </>}
+            </Col>
+          </>}
+          <Col xs style={{ marginTop: 20, }}>
+            <div id={mapElementId} style={{
+              position: 'relative',
+              width: '100%',
+              height: '260px',
+              borderRadius: 5,
+              boxShadow: '1px 1px 3px 1px #c1c1c1',
+            }}></div>
+          </Col>
+        </Row>
+      </Col>
+
+      <Col xs style={{ textAlign: 'right', margin: 8 }}>
+        <Button variant="secondary"
+          xsStyle={{ marginRight: 8 }}
+          onClick={onClose}
+        >{!loading && !recommendedAddress && address.address_line_1 ? 'Use Entered' : 'Back'}</Button>
+        <Button variant="primary"
+          onClick={() => {
+            if (!loading && !recommendedAddress && address.address_line_1) {
+              onClose();
+              return;
+            }
+
+            if (!loading && !recommendedAddress && !address.address_line_1) {
+              if (!addressLine1) {
+                return;
+              }
+              const newAddress = { ...address, address_line_1: addressLine1 };
+              setAddress?.(newAddress);
+              fetchAddress(newAddress);
+              return;
+            }
+            const newAddress = selectedValue === 'recommended' ? {
+              ...(recommendedAddress || {}),
+              address_line_1: recommendedAddress?.address_line_1 || '',
+              address_line_2: '',
+              address_line_3: '',
+            } : address;
+            onSelect?.(newAddress || {});
+            onClose();
+          }}
+        >{!loading && !recommendedAddress && address.address_line_1 ? 'Make Changes' : 'Continue'}</Button>
```

Comment:
It would be easier to read the code if the test for `!loading && !recommendedAddress && address.address_line_1` was stored in an appropriately named variable.

Reply:
I've removed the confirm address popup components since they're not being used and likely will not be used